examples/js/exporters/STLExporter.js

```diff
@@ -97,6 +99,14 @@ THREE.STLExporter.prototype = {
 
 							vector.copy( vertices[ indices[ k ] ] ).applyMatrix4( matrixWorld );
 
+              if(
```

Comment:
Looks like indentation is off -- these should be tabs.

Reply:
Sorry, my editor setup messed that up. Created a new PR
https://github.com/mrdoob/three.js/pull/16942